cmd/utils/flags.go

```diff
@@ -669,6 +670,62 @@ var (
 		Category:  flags.MiscCategory,
 	}
 
+	// Builder API settings
+	BuilderEnabled = &cli.BoolFlag{
+		Name:     "builder",
+		Usage:    "Enable the builder",
+		EnvVars:  []string{"BUILDER_ENABLED"},
+		Category: flags.BuilderCategory,
+	}
+	BuilderIgnoreLatePayloadAttributes = &cli.BoolFlag{
+		Name:     "builder.ignore_late_payload_attributes",
+		Usage:    "Builder will ignore all but the first payload attributes. Use if your CL sends non-canonical head updates.",
+		EnvVars:  []string{"BUILDER_IGNORE_LATE_PAYLOAD_ATTRIBUTES"},
+		Category: flags.BuilderCategory,
+	}
+	BuilderSecretKey = &cli.StringFlag{
+		Name:     "builder.secret_key",
+		Usage:    "Builder key used for signing blocks",
+		EnvVars:  []string{"BUILDER_SECRET_KEY"},
+		Value:    "0x2fc12ae741f29701f8e30f5de6350766c020cb80768a0ff01e6838ffd2431e11",
+		Category: flags.BuilderCategory,
+	}
+	BuilderListenAddr = &cli.StringFlag{
+		Name:     "builder.listen_addr",
+		Usage:    "Listening address for builder endpoint",
+		EnvVars:  []string{"BUILDER_LISTEN_ADDR"},
+		Value:    ":28545",
+		Category: flags.BuilderCategory,
+	}
+	BuilderGenesisForkVersion = &cli.StringFlag{
+		Name:     "builder.genesis_fork_version",
+		Usage:    "Gensis fork version.",
+		EnvVars:  []string{"BUILDER_GENESIS_FORK_VERSION"},
+		Value:    "0x00000000",
+		Category: flags.BuilderCategory,
+	}
+	BuilderBeaconEndpoints = &cli.StringFlag{
+		Name:     "builder.beacon_endpoints",
+		Usage:    "Comma separated list of beacon endpoints to connect to for beacon chain data",
```

Comment:
Is it beacon endpoint or op-node endpoint?

Reply:
its actually the payload attribute endpoint, which doesn't have BuilderGenesisForkVersion and BlockTime information